### PR TITLE
[Manual Taxes M3] Add Tracking events

### LINKED ISSUE #9793: [Manual Taxes M3] Add tracking events

### PR DESCRIPTION
**Pull request by rossanafmenezes**
### Do not merge until it's targeting trunk

Closes: #9793 

### Description

This PR add the tracking events for Milestone 3 of the Manual Taxes project. 

### Testing instructions

- Go to the Orders Tab
- Create a new order
- add a product
- click on Set New Tax Rate
- toggle the option to `Add this rate to all created orders`
- select a tax  rate
- make sure the event below is tracked with the property auto_tax_rate_enabled":true 
- untoggle the button and make sure the property is set to false

🔵 Tracked: tax_rate_selector_tax_rate_tapped, Properties: {"auto_tax_rate_enabled":true,"blog_id":221646855,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/testwoomobile.wpcomstaging.com"}

- Repeat the steps above to enable `Add this rate to all created orders`
- back to Order creation screen, click on `Edit Tax Rates Settings`
- make sure the event below is tracked when the bottom sheet is displayed

🔵 Tracked: tax_rate_auto_tax_bottom_sheet_displayed, Properties: {"blog_id":221646855,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/testwoomobile.wpcomstaging.com"}

- Repeat the steps above to display the bottom sheet (edit tax rate)
- Click on `Set a new rate for this order`
- Make sure the event below is tracked

🔵 Tracked: tax_rate_auto_tax_rate_set_new_rate_for_order_tapped, Properties: {"blog_id":221646855,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/testwoomobile.wpcomstaging.com"}

- Repeat the steps above to display the bottom sheet
- tap on `Clear Address and stop using this rate`
- make sure the event below is tracked

🔵 Tracked: tax_rate_auto_tax_rate_clear_address_tapped, Properties: {"blog_id":221646855,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/testwoomobile.wpcomstaging.com"}

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditViewModel.kt [hunk 1]
```diff
@@ -759,6 +759,7 @@ class OrderCreateEditViewModel @Inject constructor(
                 shippingAddress = EMPTY
             )
         }
+        tracker.track(AnalyticsEvent.TAX_RATE_AUTO_TAX_RATE_CLEAR_ADDRESS_TAPPED)
```
Comment (rossanafmenezes) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditViewModel.kt:762]: I haven't added a test for this given this is a private method.

Comment (samiuelson) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditViewModel.kt]: I think we should track this event in the `onStopUsingTaxRateClicked` function instead. The `clearCustomerAddresses` function is used also in `onCustomerAddressDeleted`, which is invoked in a different action.

**Review by samiuelson — COMMENTED** [review]
Looks good overall 👍🏼 

I've added a comment on one event. Let me know what you think.

**Comment by rossanafmenezes** [comment]
Thank you @samiuelson (for some reason I can't reply to your comment anymore) I have fixed what you mentioned here https://github.com/woocommerce/woocommerce-android/pull/9795/commits/392e0d8c8193e091795266e1c696521d9c70065b

**Review by samiuelson — APPROVED** [review]
LGTM 🎉

**Pull request merged**